tests/Feature/Api/Location/DeleteControllerTest.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Feature\Api\Location;
+
+use App\Domain\Locations\Models\Location;
+use App\Domain\Trips\Models\Trip;
+use App\Models\User;
+use Laravel\Sanctum\Sanctum;
+use PHPUnit\Framework\Attributes\Group;
+use PHPUnit\Framework\Attributes\Test;
+use Tests\TestCase;
+
+#[Group("Locations")]
+class DeleteControllerTest extends TestCase
+{
+    #[Test]
+    public function it_cannot_delete_a_loccation_if_not_logged_in(): void
+    {
+        $location = Location::factory()->create();
+
+        $this->deleteJson(route('api.locations.delete', $location->id))->assertUnauthorized();
+    }
+
+    #[Test]
+    public function it_cannot_delete_a_location_for_a_location_that_doesnt_exist(): void
+    {
+        $this->deleteJson(route('api.locations.delete', 'asdadsa'))->assertUnauthorized();
```

Comment:
I think this test is failing first because you aren't logged in, should we add login here and then assert that non existent location cannot be deleted?

Reply:
Good spot i've updated this to now be a not found and logged in a user